### PR TITLE
Mention autoescape=true deprecation (issue #2386)

### PR DESCRIPTION
**Pull request by vphantom**
Per #2386, mention the deprecation of true in favor of an explicit "html".

### CODE REVIEW

**Comment by SpacePossum** [comment]
>As of Twig 1.x

please change to:

>As of Twig 1.21

File: doc/deprecated.rst [hunk 1]
```diff
@@ -206,6 +206,9 @@ Miscellaneous
   ``Twig_TemplateInterface::getEnvironment()`` are deprecated and will be
   removed in 2.0.
 
+* As of Twig 1.x, setting the environment option ``autoescape`` to ``true`` is
```
Comment (stof) [inline doc/deprecated.rst]: 1.8 rather than 1.x

**Comment by vphantom** [comment]
First time amending a PR… It seems to have worked as intended.

**Comment by SpacePossum** [comment]
Looking good :). :+1:

**Comment by stof** [comment]
👍

**Comment by fabpot** [comment]
Thank you @vphantom.

**Pull request closed without merging**